backend/post-service/src/main/java/com/speakapp/postservice/repositories/PostReactionRepository.java

```diff
@@ -15,6 +15,9 @@
 public interface PostReactionRepository extends JpaRepository<PostReaction, UUID> {
 
     List<PostReaction> findAllByPost(Post post);
+
+    PostReaction findByPostAndUserId(Post post, UUID userId);
```

Comment:
Arek dodał spoko query pod spodem, do tego co chcesz zrobić, zerknij sobie na jego kod. Na przyszłość jak chcesz dostać jeden obiekt z bazy danych to pamiętaj o opakowaniu w Optional<>, wymusza to na korzystającym z tej metody obsługę sytuacji gdy obiektu nie ma w tabeli - w następnym komie zobaczysz o czym mówiłem.